projects/packages/forms/src/contact-form/class-contact-form.php

```diff
@@ -133,8 +133,9 @@ public function __construct( $attributes, $content = null, $set_id = true ) {
 		$this->is_response_without_reload_enabled = apply_filters( 'jetpack_forms_enable_ajax_submission', true );
 
 		// Set up the default subject and recipient for this form.
-		$default_to      = self::get_default_to( $this->current_post ? $this->current_post->post_author : null );
-		$default_subject = self::get_default_subject( $attributes );
+		$post_author_id  = self::get_post_property( $this->current_post, 'post_author' );
+		$default_to      = self::get_default_to( $post_author_id );
+		$default_subject = self::get_default_subject( $attributes, $this->current_post );
```

Comment:
I notice above we set $post_author_id, which seems like an addition unrelated the original issue. Just wondering why that's added.

Reply:
`$post_author_id` is used in the line below to define `$default_to`, since `current_post->post_author` could end up triggering the warning.